books/xdoc/base.lisp

```diff
@@ -118,7 +120,18 @@
                           (cons :parents parents)
                           (cons :short short)
                           (cons :long long)
-                          (cons :from bookname)))
+                          ;; :from is the book where the doc is defined,
+                          ;; (usually relativized to acl2/books and with
+                          ;; :DIR :SYSTEM added)
+                          (cons :from bookname)
+                          ;; :code-from is specified by the xdoc writer,
+                          ;; and is not currently not modified/normalized
+                          ;; so when specified, the writer should self-normalize
+                          ;; it to a form similar to the :from (for now at
+                          ;; least)
+                          ; [Pending change]
+                          ; (cons :code-from code-source)
```

Comment:
why is this still commented out? Shouldn't this change be tested with this uncommented?

Reply:
Because if the xdoc table entry changes now, all the consumers will need to be OK with that, and I was probably missing one when I tried it before since I got errors that were hard to debug.  I'm trying for a staged approach to make it easier.